modin/pandas/groupby.py

```diff
@@ -1114,9 +1114,20 @@ def rolling(self, *args, **kwargs):
     def hist(self):
         return self._default_to_pandas(lambda df: df.hist())
 
-    def quantile(self, q=0.5, interpolation="linear"):
-        # TODO: pandas 1.5 now supports numeric_only as an argument
+    def quantile(self, q=0.5, interpolation="linear", numeric_only=no_default):
         # TODO: handle list-like cases properly
+        if numeric_only is no_default:
+            numeric_only = True
```

Comment:
pandas logic is much more complicated than this, though...

https://github.com/pandas-dev/pandas/blob/2e218d10984e9919f0296931d92ea851c6a6faf5/pandas/core/groupby/groupby.py#L1321-L1372

Reply:
True, but the 1.5 docs don't have any mention of a SeriesGroupBy.quantile method, and the DataFrameGroupby.quantile method defaults to true for numeric_only.

Reply:
Yes, we are operating on this for quantile (the API was not matching): https://pandas.pydata.org/pandas-docs/version/1.5/reference/api/pandas.core.groupby.DataFrameGroupBy.quantile.html